frontend-react/src/components/MessageTracker/MessageReceivers.test.tsx

```diff
@@ -13,79 +14,65 @@ describe("MessageReceivers component", () => {
-        expect(screen.getByText(/Receiver Name/)).toBeInTheDocument();
+        expect(screen.getAllByText(/Name/)[0]).toBeInTheDocument();
         expect(screen.getByText(/md-phd/)).toBeInTheDocument();
-        expect(screen.getByText(/Receiver Service/)).toBeInTheDocument();
+        expect(screen.getByText(/Service/)).toBeInTheDocument();
         expect(screen.getByText("elr")).toBeInTheDocument();
-        expect(screen.getByText(/Incoming Report Id/)).toBeInTheDocument();
+        expect(screen.getByText("Date")).toBeInTheDocument();
+        expect(
+            screen.getByText(/09\/28\/2022, 07:55:12 PM/)
+        ).toBeInTheDocument();
+        expect(screen.getByText(/Report Id/)).toBeInTheDocument();
         expect(
             screen.getByText(/578eae4e-b24d-45aa-bc5c-4d96a0bfef96/)
         ).toBeInTheDocument();
-        expect(screen.getByText(/Incoming File Name/)).toBeInTheDocument();
-        expect(screen.getByText(/fl-covid-19.hl7/)).toBeInTheDocument();
-        expect(screen.getByText(/Incoming File URL/)).toBeInTheDocument();
+        expect(screen.getByText(/Main/)).toBeInTheDocument();
+        expect(screen.getAllByText(/BATCH/)[0]).toBeInTheDocument();
+        expect(screen.getByText(/Sub/)).toBeInTheDocument();
+        expect(screen.getByText(/ignore.HL7_NULL/)).toBeInTheDocument();
+        expect(screen.getByText(/File Name/)).toBeInTheDocument();
         expect(
-            screen.getByText("https://azurite:10000/devstoreaccount1.csv")
+            screen.getByText(
+                /tx-covid-19-4b3c73df-83b1-48f9-a5a2-ce0c38662f7c-20230203182255.internal.csv/
+            )
         ).toBeInTheDocument();
-        expect(screen.getByText("Date/Time Submitted")).toBeInTheDocument();
-        expect(screen.getByText(/September 28 2022/)).toBeInTheDocument();
+
         expect(screen.getByText("Transport Results")).toBeInTheDocument();
         expect(screen.getByText(/Transport result/)).toBeInTheDocument();
 
-        expect(screen.getByText(/Quality Filters/)).toBeInTheDocument();
         expect(
-            screen.getByText(
-                "For ak-phd.elr, filter hasValidDataFor[patient_dob] filtered out item Alaska1"
-            )
-        ).toBeInTheDocument();
+            screen
+                .getByRole("table")
+                .getElementsByClassName("message-receiver-break-word")[0]
+        ).toHaveTextContent("578eae4e-b24d-45aa-bc5c-4d96a0bfef96");
+
+        await userEvent.click(screen.getByText(/^Report Id$/));
         expect(
-            screen.getByText(
-                "For ak-phd.elr, filter isValidCLIA[testing_lab_clia, reporting_facility_clia] filtered out item Alaska1"
-            )
-        ).toBeInTheDocument();
+            screen
+                .getByRole("table")
+                .getElementsByClassName("message-receiver-break-word")[0]
+        ).toHaveTextContent("400eae4e-b24d-45aa-bc5c-4d96a0bfef96");
     });
```

Comment:
This test is showing that upon click of `Report Id`, the order of the data has changed from:

1st: `578eae4e-b24d-45aa-bc5c-4d96a0bfef96`
2nd: `400eae4e-b24d-45aa-bc5c-4d96a0bfef96`

TO:

1st: `400eae4e-b24d-45aa-bc5c-4d96a0bfef96`
2nd: `578eae4e-b24d-45aa-bc5c-4d96a0bfef96`

Reply:
I added a comment about this [here](https://github.com/CDCgov/prime-reportstream/pull/8295/files#r1114978097), but I think we should break out the sorting logic from the component body so we can test the sorting proper versus the component proper.  If this test fails, there could be a number of reasons:
- Something failed to render
- Class names changed
- Role changed
- Fake data changed
- Sorting logic changed
- ...probably a bunch of other things

If we broke out the sorting logic (and data mapping logic for that matter!) and saw failing tests for that, we'd know exactly what's failing rather than having to debug the rendering in addition to the sorting.